src/platforms/workable/index.ts

```diff
@@ -0,0 +1,23 @@
+import { client } from '@/platforms/workable/client';
+import boxIcon from '@/platforms/workable/logos/box';
+import fullIcon from '@/platforms/workable/logos/full';
+import { auth, platform } from '@/sdk';
+
+export default platform('workable', {
+  auth: auth.apiToken(),
```

Comment:
hmmm noticed that it didn't add the questions for the domain for apitoken... will need to add that, only have it on oauth rn